### PR TITLE
fix: preserve original image url

### PR DESCRIPTION
**Pull request by artificial-intelligence**
Resolves: https://github.com/osism/issues/issues/913

Signed-off-by: Sven Kieske <kieske@osism.tech>

### CODE REVIEW

File: openstack_image_manager/update.py [hunk 1]
```diff
@@ -243,7 +243,8 @@ def update_image(
     minio_bucket = str(minio_bucket)
     new_url = f"https://{minio_server}/{swift_prefix}{minio_bucket}/{shortname}/{current_version}-{shortname}.{format}"
     logger.info(f"New URL is {new_url}")
-    image["versions"][0]["url"] = new_url
+    image["versions"][0]["mirror_url"] = new_url
+    image["versions"][0]["url"] = latest_url
```
Comment (mbuechse) [inline openstack_image_manager/update.py:247]: Not sure about this line. I think we decided that `current_url` be used here, and this is what l. 234 already does.

See https://github.com/osism/openstack-image-manager/issues/639 for reference.

Comment (artificial-intelligence) [reply openstack_image_manager/update.py]: thanks for the review, should be hopefully fixed now.

Comment (mbuechse) [reply openstack_image_manager/update.py]: This should work, but I still think that l. 234 already takes care of "url" (in combination with l. 237), so l. 247 should be redundant. But I don't know for sure, I mean, @berendt must have had some reason to alter this in the first place...

**Comment by berendt** [comment]
This will not fix the issue, this will make the generated image definitions unusable.

**Pull request closed without merging**